### PR TITLE
Save skybox

### PR DESCRIPTION
**Pull request by Ysabz**
### Summary 
This PRs adds the functionality to save the skybox for each experience that will be loaded when editor spins up. Also, small cleanup and UI fixes
### How to test?
_as before + this time change the skybox of your experience and save and comeback to it. the new skybox will be loaded in the editor 
### Checks 
- [ ] Tested Changes
- [ ] Stakeholder Approval

### CODE REVIEW

**Review by mckenzietsean — APPROVED** [review]
Saving the skybox works and loads in quicker than expected. All is good.

File: src/components/Editor/TextEditor.tsx [hunk 1]
```diff
@@ -13,11 +13,11 @@ const TextDiv = styled.div`
   background: #3f3d56;
   display: flex;
   width: inherit;
-  height:inherit;
+  height: inherit;
   flex-direction: column;
   border-radius: 15px;
```
Comment (shushengfuqin) [inline src/components/Editor/TextEditor.tsx:18]: is it possible to keep using em instead of px

File: src/components/Editor/TextEditor.tsx [hunk 2]
```diff
@@ -60,28 +60,26 @@ const TextEditor = (): JSX.Element => {
...
         }}
       >
-        <textarea 
+        <textarea
           style={{
             background: "white",
             borderRadius: "5px",
             width: "100%",
-            height:"100%",
+            height: "100%",
```
Comment (shushengfuqin) [inline src/components/Editor/TextEditor.tsx:78]: when i was working on it, i don't really understand the meaning of 100%. is there any way to use inherit in this type of styling?

**Review by shushengfuqin — APPROVED** [review]

File: src/components/Editor/TextEditor.tsx (hunk 2 above)
Comment (Ysabz) [reply src/components/Editor/TextEditor.tsx:78]: the problem with inherit is that it gets the width of the parent so the child component will go out of parent. 100% tells the component to take the available spacing but be contained in parent

File: src/components/Editor/TextEditor.tsx (hunk 1 above)
Comment (Ysabz) [reply src/components/Editor/TextEditor.tsx:18]: for border radius, px should be fine but I used em for all the other sizings

**Pull request merged**